package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@the-collab-lab/shopping-list-utils": "^1.0.1",
+    "@the-collab-lab/shopping-list-utils": "^1.0.3",
```

Comment:
Certainly fine to bump this up but 1.0.3 was already in use in this project. The `^1.0.1` says "I want anything from 1.0.1 and higher, up until but not including 1.2.0. The actual version in use is set in package-lock.json.